interpreter/code_interpreters/languages/python.py

```diff
@@ -4,6 +4,8 @@
 import re
 
 class Python(SubprocessCodeInterpreter):
+    file_extension = "py"
```

Comment:
These `file_extension`s aren't strictly necessary, but it felt nicer when generating the temporary file for semgrep to scan to do it with the appropriate extension and could come in handy for some future ideas - like adding some functionality to directly export generated code into a file via a some `%magic` command without needing to have another exchange with Open Interpreter to get it to create some code to pipe the other code into the file.

In the case of something like the JavaScript language configuration this could get a little more complicated since they could also be modules that use the `mjs` extension, but for the purposes it's being used for right now - a naive extension suffices.